README.md

```diff
@@ -26,7 +26,13 @@ We organize two meetups every month: one for talks, one for hacks.
 
 ### Presentation Nights
 
-Every 2nd Thursday we put on a presentation night at [Vacasa](https://www.vacasa.com/contact-us/) where the goal is to provide high-quality presentations from people in the Node community locally and abroad. At the present we typically have local speakers present in person, and other geographically distributed speakers present remotely via talky.io.
+Every 2nd Thursday we put on a presentation night at [Vacasa](https://www.vacasa.com/contact-us/) where the goal is to provide [high-quality presentations](https://www.youtube.com/channel/UCI8MIw5A7ALtIvNHsrYJbjg) from people in the Node community locally and abroad. At the present we typically have local speakers present in person, and other geographically distributed speakers present remotely via [talky.io](https://talky.io/).
+**<h6>Recent Presentation</h6>**
```

Comment:
were you trying to make the header here bolder?

here's the style guide for how everything renders with the style library we're using:

https://ungoldman.github.io/style.css/guide.html

Reply:
@ungoldman locally this produced the bolder header I was looking for so I went with it, but yeah it won't perform the same way cross-platform and moving forward I’ll stick to the styleguide–avoiding odd combinatorial styling choices like this. 👍